web/settings.py

```
@@ -163,6 +165,7 @@
                 "django.contrib.auth.context_processors.auth",
                 "django.contrib.messages.context_processors.messages",
                 "web.context_processors.last_modified",
+                "social_django.context_processors.backends"
```

Ruff (0.12.2)

Trailing comma missing

Add trailing comma

(COM812)